andela/valhalla-rc
# #159218128 display username correctly

#### What does this PR do?
Allows users to see a username after successful registration.

#### Description of Task to be completed?
- extract username from email address
- show extracted username on navbar and profile page
- correctly route profile link from user profile

#### How should this be manually tested?
- Clone the repo, run ``meteor npm install`` and visit `localhost:3000`
- register as a new user and check the username on the navigation bar and on the profile page

#### Any background context you want to provide?
- Previously, when a user registers on the platform, he/she has a username of `guest` until the profile details have been filled.
- With this pull request, the user is given a temporary username gotten from the email address provided

#### What are the relevant pivotal tracker stories?
[Finishes #159218128](https://www.pivotaltracker.com/n/projects/159218128)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-07-23 at 5 17 09 pm" src="https://user-images.githubusercontent.com/32115854/43089484-a7655b6e-8e9c-11e8-942d-f6f200c8d721.png">
<img width="1440" alt="screen shot 2018-07-23 at 5 17 43 pm" src="https://user-images.githubusercontent.com/32115854/43089513-b96c093e-8e9c-11e8-8010-0343be69d4f0.png">

#### Questions:
N/A
